move-mutation-test/src/lib.rs

```diff
@@ -170,19 +144,14 @@ pub fn run_mutation_test(
             }
 
             benchmark.start();
-
-            // No need to fetch latest deps again.
-            let skip_fetch_deps = true;
-            // No need to print anything to the screen, due to many threads, it might be messy and slow.
-            let mut error_writer = std::io::sink();
-            let result = run_tests(&test_config, &outdir, skip_fetch_deps, &mut error_writer);
+            let result = run_tests_on_mutated_code(test_config, &outdir);
             benchmark.stop();
 
             let mutant_status = if let Err(e) = result {
                 trace!("Mutant killed! Unit test failed with error: {e}");
                 MutantStatus::Killed
             } else {
-                trace!("Mutant {} hasn't been killed!", mutant_file.display());
+                info!("Mutant {} hasn't been killed!", mutant_file.display());
```

Comment:
I was wondering if changing the log level here to `info` shouldn't introduce also a change to mutant killed - as there is still a `trace` level.  And I'm still hesitating.

From one point of view, it would be nice to have that info on the same level, but on the other hand, not killed mutants are more important that the killed ones 😄

Reply:
Because then you don't even have to open the `report.txt`, it's all visible there in the output if the `RUST_LOG` is enabled